packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -79,8 +80,12 @@ export const selectFirstAvailableDevice = () => () => {};
 // export const handleDeviceEvent = (device: Device) => (dispatch: Dispatch, getState: GetState) => {
 export const handleDeviceConnect = (device: Device) => (dispatch: Dispatch, getState: GetState) => {
     const selected = getState().suite.device;
+    const { app } = getState().router;
     if (!selected) {
         dispatch(selectDevice(device));
+        if (device.mode === 'initialize' && app !== 'onboarding') {
+            goto('/onboarding');
```

Comment:
please use `getRoute` from utils/suite/router.ts

Reply:
Ok, fixed. Check it now.